R/analyse_oneshot_spatial.R

```diff
@@ -10,9 +14,17 @@
 #' @examples
 #' if(interactive()) {
 #'   analyse_oneshot_spatial (
-#'     raster_files = c("r1.tif", "r2.tif"),
-#'     calculations = c("calc_endemism_central", "calc_richness", "calc_pd"),
-#'     tree = some_phylo_tree
+#'     raster_files             = c("r1.tif", "r2.tif"),
+#'     r_data                   = list(
+#'                                    '250:250' = list (r1 = 13758, r2 = 13860, r3 = 13727),
+#'                                    '250:750' = list (r1 = 11003, r2 = 11134, r3 = 11279),
+#'                                    )
+#'     spreadsheet_data         = r[12].xlsx
```

Comment:
This and the next two lines should spell out the file names, similar to the ```raster_files``` example above.  Trailing comma is also needed, e.g.:  ```c("r1.xlsx", "r2.xlsx"),```

Reply:
Basically it needs to be valid R code.

Reply:
Added file names and trailing commas.